docker/docker-compose-local.yml

```diff
@@ -21,6 +21,10 @@ services:
         environment:
             - LOAD_EX=n
             - EXECUTOR=Local
+            - AWS_DEFAULT_REGION=us-west-2
+            - AWS_REGION=us-west-2
+            - AWS_ACCESS_KEY_ID=${AWS_ACCESS_KEY_ID}
+            - AWS_SECRET_ACCESS_KEY=${AWS_SECRET_ACCESS_KEY}
```

Comment:
I'm not sure if this is **the best** way but it worked in my local environment. I researched a bit to get to this point and it seems that using environment variables here are OK. Would like to understand best practices.